spec/openfeature/example_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+RSpec.describe "README example" do
+  before do
+    @readme_md = Pathname.new("../../../README.md").expand_path(__FILE__)
```

Comment:
Is it possible to get the root directory here

Reply:
Do you mean like a helper? There isn't an equivalent of `Rails.root` for gems or specs unless you implement it yourself